scripts/clarity-icons-svg.js

```diff
@@ -12,8 +17,17 @@ const SHAPE_SETS = [
 ];
 
 writeSVGIcons(SHAPE_SETS, () => {
-    shell.exec("cd dist/clarity-icons/shapes; zip -r all-shapes.zip ./**/*");
-    SHAPE_SETS.forEach((setName) => {
-        shell.exec(`cd dist/clarity-icons/shapes; zip -r ${setName}.zip ./${setName}/*; rm -r ./${setName}`);
-    });
+    if (isWin) {
```

Comment:
I like the use of `bestzip`. I think it can help us make it even simpler because the `shell.pushd` command and `bestzip` seem to be cross platform compatible. I was able to build on both my windows VM and on my OSX machine with a few small tweaks to your code. 

From the `bestzip` docs, 

> This module provides a bestzip command that calls the native zip command if available and otherwise falls back to a Node.js implementation.

I was able to take out the check for windows and package up the icons like this. It built on my windows VM and on OSX with this:

```
writeSVGIcons(SHAPE_SETS, () => {
    shell.pushd("./dist/clarity-icons/shapes");
    shell.exec(`bestzip all-shapes.zip ${SHAPE_SETS.join(' ')}`);
    SHAPE_SETS.forEach((setName) => {
        shell.exec(`bestzip ${setName}.zip ${setName}/*`);
    shell.rm('-rf', `${setName}\`);
    });
});
```

That enables us keep the build script as simple as possible.